### ISSUE TITLE
How to store data over 256M?

### ISSUE THREAD

**Issue by SmarterEric** [filecoin-project/go-filecoin#3533]
I have stored many pieces data which was 256MB, and some of them have been reached 'complete' status, but the miner power was always 268.4 MB, never increased up, also the Filecoin Network stats only show 268.4 MB. I don't why some other miners can increase their power, like 'FILPool', which Proven Storage was 2.3TB.

I tried store a data which size was 400MB, but it will returned a Error when I stored it into filecoin.
、、、
~/filecoin/./go-filecoin client import write.mp4    ~/filecoin/./go-filecoin client propose-storage-deal t2n6mbpnld7b3sn7t2gagi6yxw2btbzo5zcqu5ara QmXYzArFLEJmA8amuRdj42czXLiXU297KwduDmS7QB3raJ 000 20160
Error: piece is 502902764 bytes but sector size is 266338304 bytes
[japan@VM_0_16_centos ~]$
、、、

Pls give me a method how to increase up my miner storage power over 268.4 MB, thanks.

**Comment by SmarterEric** [comment]
I have stored many pieces data which was 256MB, and some of them have been reached 'complete' status, but the miner power was always 268.4 MB, never increased up, also the Filecoin Network stats only show 268.4 MB. I don't konw why some other miners can increase their power, like 'FILPool', which Proven Storage was 2.3TB.

I tried store a data which size was 400MB, but it will returned a Error when I stored it into filecoin.
、、、
~/filecoin/./go-filecoin client import write.mp4 ~/filecoin/./go-filecoin client propose-storage-deal t2n6mbpnld7b3sn7t2gagi6yxw2btbzo5zcqu5ara QmXYzArFLEJmA8amuRdj42czXLiXU297KwduDmS7QB3raJ 000 20160
Error: piece is 502902764 bytes but sector size is 266338304 bytes
[japan@VM_0_16_centos ~]$
、、、

Pls give me a method how to increase up my miner storage power over 268.4 MB, thanks.
